bbc/simorgh
# South Asia sites: Navigation and Features box updates after Tipo migration

Resolves JIRA n/a

Upated the navigation bar of Bengali, Sinhala, and Urdu to reflect changes required by editorial after the sites have migrated to Tipo. Fixed/Updated the Bengali and Marathi headings of the Features box on articles


Code changes
======

- Removed election topic link from Navigation section of src/app/lib/config/services/urdu.ts
- Removed link to FIX and added links to Art and Video topics in the navigation section of src/app/lib/config/services/sinhala.ts
- Added Most read lnk to the Navigation section of src/app/lib/config/services/bengali.ts
- Replaced Incorrect English text in featuresAnalysisTitle: on  src/app/lib/config/services/marathi.ts
- Replaced text in featuresAnalysisTitle: on  src/app/lib/config/services/bengali.ts so that it matches the new section heading on the front page

Testing
======
1. Open bbc.com/urdu third link on the nav bar, there should be no link to الیکشن 2023 (https://www.bbc.com/urdu/topics/cynd7qxprq0t)
2. Open bbc.com/bengali, last link on the nav bar should point to /bengali/popular/read
3. Open bbc.com/sinhala, last two links on the nav bar should point to /sinhala/topics/crldzm9n2lnt and /sinhala/topics/c7zp5zxk8jxt
4. Open an article on bbc.com/marathi. On the right hand column the heading of the Features box under Top stories should be 'बीबीसी मराठी स्पेशल' not 'Features'
5. Open an article on bbc.com/bengali. On the right hand column the heading of the Features box under Top stories should be 'নির্বাচিত খবর'
